requirements.txt

```diff
@@ -17,6 +16,8 @@ Sphinx==1.3.1
 oic==0.7.6
 pyOpenSSL==0.15.1
 lxml==3.4.4
+protobuf==3.0.0b1.post1
+datetime
```

Comment:
Mostly. There's an extended DateTime package (https://pypi.python.org/pypi/DateTime/4.0.1) but I don't think that's used here, so I'll remove it.